diktat-rules/src/test/resources/test/paragraph2/kdoc/OrderedTagsAssertionTest.kt

```diff
@@ -0,0 +1,13 @@
+package com.saveourtool.save.reporter.json
+
+/**
+ * Reporter that produces a JSON report as a [Report]
+ *
+ * @property out a sink for output
+ *
```

Comment:
we only fix this line here?
Which rule caused an issue?

Reply:
KDOC_WRONG_TAGS_ORDER property and param in a wrong order + new line between them